### ISSUE TITLE
Image refresh for debian-testing

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/bots#335]
Image refresh for debian-testing
 * [ ] FAIL: image-refresh debian-testing

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-01.
Log: https://209.132.184.41:8493/logs/image-refresh-335-20191215-223232/

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-10.
Log: https://209.132.184.41:8493/logs/image-refresh-335-20191215-223131/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-335-20191215-223131/

**Comment by martinpitt** [comment]
Apparently we need to apply a bigger hammer to avoid [conffile prompts](https://209.132.184.41:8493/logs/image-refresh-335-20191215-223131/log):
```
Setting up grub-cloud-amd64 (0.0.5) ...

Configuration file '/etc/default/grub'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** grub (Y/I/N/O/D/Z) [default=N] ? dpkg: error processing package grub-cloud-amd64 (--configure):
 end of file on stdin at conffile prompt
```

**Comment by martinpitt** [comment]
Fixed in #389
